lib/js/components/SurveyQuestions/SurveyQuestionScale/SurveyQuestionScale.spec.ts

```diff
@@ -29,10 +28,6 @@ describe('SurveyQuestionScale', () => {
 			slots: {
 				explanation,
 			},
-			stubs: {
-				SurveyToggle,
-				Card,
```

Comment:
Alternatively we could use string stubs, something like `<div><slot/><slot name="footer"/></div>`. Using `mount` was easier.